rdkit/Chem/Pharm3D/test_list.py

```diff
@@ -1,6 +1,7 @@
 tests = [
-  ("python", "EmbedLib.py", {}),
+  #   ("python", "EmbedLib.py", {}),
```

Comment:
Given that file still contains doctests (and should still contain doctests), they should be run. Please turn this back on.